makefile

```diff
@@ -24,41 +25,31 @@ run-ra-dev: ## run the react-admin server.
 ########
 
 run: ## fresh run of all you need to use the app
-	make run-postgrest-docker && make create-model && make populate-db && make run-ra-dev
+	make start-supabase && make supabase-migrate-database && make supabase-populate
 
 stop: ## stop the docker
-	make stop-postgrest-docker
-
-## Docker - postrgrest / postgres
-#################################
-
-run-postgrest-docker: ## run the postgrest docker-compose containing 2 dockers: postgres (db) & postgrest (api). The postgres docker start with a ready to use empty db.
-	docker-compose --env-file .env -f docker-compose.yml up --build -d
-
-stop-postgrest-docker: ## stop the postgrest docker-compose
-	docker-compose --env-file .env -f docker-compose.yml down
+	make stop-supabase
 
-## Database
-###########
+## Supabase (postrgrest / postgres)
+###################################
 
-connect-db: ## connect into the postres docker.
-	docker exec -it connect-four-reboot-admin-database-postgres-1 sh -c 'psql -U $(DATABASE_USER) -d $(DATABASE_NAME)'
+start-supabase: ## start supabase locally
+	npx supabase start
 
-create-db: ## initialize an empty ready to use db inside the docker - use it only after drop.
-	docker exec -i connect-four-reboot-admin-database-postgres-1 sh -c 'psql -U $(DATABASE_USER) -c "CREATE DATABASE $(DATABASE_NAME);"'
+stop-supabase: ## stop supabase locally
+	npx supabase stop
 
-drop-db: ## drop the postgres db inside the docker.
-	docker exec -i connect-four-reboot-admin-database-postgres-1 sh -c \
-'psql -U postgres -c "SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = '\''connect_four_reboot_admin'\'';"' && docker exec -i connect-four-reboot-admin-database-postgres-1 sh -c 'psql -U $(DATABASE_USER) -c "DROP DATABASE IF EXISTS $(DATABASE_NAME);"'
+supabase-migrate-database: ## apply the migrations to the database
+	npx supabase migration up
 
-create-model: ## create the connect-four-reboot-admin tables.
-	docker exec connect-four-reboot-admin-database-postgres-1 sh -c 'psql -U $(DATABASE_USER) -d $(DATABASE_NAME) -f /scripts/create_model.sql'
+supabase-reset-database: ## reset (and clear!) the database
+	npx supabase db reset
 
-populate-db: ## populate database with fake values
+supabase-populate: ## populate database with fake values
 	npx tsx tools/populateDbWithFakeData.ts
 
-reset-db: 
-	make drop-db && make create-db && make create-model && make populate-db
+supabase-connect: ## connect into the supabase postgres docker.
+	docker exec -it ${SUPABASE_DOCKER_NAME} sh -c 'psql -U $(SUPABASE_USER) -d $(SUPABASE_DBNAME)'
```

Comment:
nitpick (non-blocking): it doesn't hurt to have this target, but it's not really necessary anymore since, now, you can easily see what's inside the database (and much more) from the Supabase dashboard at http://localhost:54323/